### ISSUE TITLE
Is ChromBPNet trained with additional data?

### ISSUE THREAD

**Issue by gonzalobenegas** [kundajelab/DART-Eval#6]
Thanks for your work, excited to see the paper out!

From reading the paper I understand that for Tasks 4 and 5, only the predicted total reads are used from ChromBPNet. However, you describe that ChromBPNet was also trained on profile prediction.
> The count prediction was trained using mean squared error loss, while the profile head was trained using log-likelihood loss based on a multinomial distribution.

> We utilized the same training setup for our probing and fine-tuning models so that inputs and labels were identical to
those for ChromBPNet.

Would it be fair to say that ChromBPNet is given additional, profile-level labels?

**Comment by amanpatel101** [comment]
Hey Gonzalo - my apologies for the extremely late response. I totally missed this until I was just alerted about it by someone else. 

I think your statement is fair. ATAC-seq data comes in the form of read counts per base. If you add them all together, you can get the total counts over the region, which is what all our models are trained on. If you instead normalize the per-base count values, you can get a probability distribution over the region, which is the profile distribution that ChromBPNet is trained on. We only compared the ChromBPNet counts head to the language models, but the two heads can sometimes provide complementary insights.

**Comment by gonzalobenegas** [comment]
Thank you for your thoughts! If you ever implement the additional loss in this repo, let me know!
